packages/retail-ui-extensions/src/components/Screen/Screen.ts

```diff
@@ -1,8 +1,20 @@
 import {createRemoteComponent} from '@remote-ui/core';
 
+export interface ScreenSearchBarProps {
+  initialValue?: string;
+  onTextChange?: (value: string) => void;
+  onSearch: (value: string) => void;
+  onFocus?: () => void;
+  visible?: boolean;
```

Comment:
This is a new prop I added. Let me know if you think it's useful.

Reply:
Can we double check the naming convention we use for these booleans with core-app? ie `isVisible` vs `visible`. Just so we get it right the first time

Reply:
@heltisace could you please clarify what this `visible` prop will do? Is this the prop that we discussed that should make the `SearchBar` smaller?